KPP/Hg/gckpp_Integrator.F90

```diff
@@ -494,7 +494,7 @@ SUBROUTINE ros_ErrorMsg(Code,T,H,IERR)
     CASE (-6)
       PRINT * , '--> No of steps exceeds maximum bound'
     CASE (-7)
-      PRINT * , '--> Step size too small: T + 10*H = T', &
+      PRINT * , '--> Step size too small: T + 0.1*H = T', &
```

Comment:
Is this an intended update? I didn't see any mention of it

Reply:
Note: The original error message was incorrect and fixed in the latest version of KPP. This can be resolved.